config/schema.yaml

```diff
@@ -0,0 +1,330 @@
+00documentation:
+  default:
+  - Precedence order (https://www.npmjs.com/package/convict)
+  - .
+  - When merging configuration values from different sources,
+  - Convict follows precedence rules. The order, from lowest to highest, for
+  - config.loadFile(file) and config.load(json) is
+  - .
+  - Default value
+  - File or json set in function argument
+  - Environment variables (only used when env property is set in schema)
+  - Command line arguments (only used when arg property is set in schema)
+  - .
+  - For easy json/yaml conversion: https://www.json2yaml.com/
+  - .
+  - NOTE: Please be careful when chosing defaults for undefined
+  - environment variables. "undefined" is recommended
+  - .
+  - The defaults for all variables below are currently for development builds.
+  doc: 00documentation
+  env: 00DOCUMENTATION
+admin_email_data_export:
+  default: ''
+  doc: admin_email_data_export
+  env: ADMIN_EMAIL_DATA_EXPORT
+admin_email_data_export_test:
+  default: ''
+  doc: admin_email_data_export_test
+  env: ADMIN_EMAIL_DATA_EXPORT_TEST
+admin_email_email_test:
+  default: ''
+  doc: admin_email_email_test
+  env: ADMIN_EMAIL_EMAIL_TEST
+admin_emails:
+  default: []
+  doc: admin_emails
+  env: ADMIN_EMAILS
+admin_uids:
+  default: []
+  doc: admin_uids
+  env: ADMIN_UIDS
+akismet_antispam_api_key:
+  default: ''
+  doc: akismet_antispam_api_key (ex. a1a11111aa11)
+  env: AKISMET_ANTISPAM_API_KEY
+  format: String
+akismet_root_url:
+  default: https://pol.is
+  doc: akismet_root_url
+  env: AKISMET_ROOT_URL
+aws_access_key_id:
+  default: undefined
+  doc: aws_access_key_id
+  env: AWS_ACCESS_KEY_ID
+  sensitive: true
+  tag: api
+aws_region:
+  default: us-east-1
+  doc: aws_region
+  env: AWS_REGION
+aws_s3_api_version:
+  default: 2006-03-01
+  doc: aws_s3_api_version
+  env: AWS_S3_API_VERSION
+aws_secret_access_key:
+  default: undefined
+  doc: aws_secret_access_key
+  env: AWS_SECRET_ACCESS_KEY
+  sensitive: true
+aws_ses_api_version:
+  default: 2010-12-01
+  doc: aws_s3_api_version
+  env: AWS_S3_API_VERSION
+backfill_comment_lang_detection:
+  default: false
+  doc: backfill_comment_lang_detection
+  env: BACKFILL_COMMENT_LANG_DETECTION
+cache_math_results:
+  default: false
+  doc: cache_math_results
+  env: CACHE_MATH_RESULTS
+client_participation_port:
+  default: 5001
+  doc: Note that this must match the participation client port specified in polisServer
+    instance
+  env: CLIENT_PARTICIPATION_PORT
+database_for_reads_name:
+  default: postgres://postgres:oiPorg3Nrz0yqDLE@postgres:5432/polis-dev
```

Comment:
The database names as written here hard code the password, so these should be marked as `sensitive`, and better document both here and in any pertinent docker configuration that default passwords should not be used for production deployments.

Reply:
Actually... this is wrong. I think the `database_for_reads_name` should point not to a url, but to an env/config variable that points to a url (IIRC).